README.md

```diff
@@ -9,57 +9,88 @@ play-frontend-hmrc is a Scala Twirl implementation of
-- [Adding accessible-autocomplete CSS and Javascript](#adding-accessible-autocomplete-css-and-javascript)
-- [Adding your own SASS compilation pipeline](#adding-your-own-sass-compilation-pipeline)
-- [Play Framework and Scala compatibility notes](#play-framework-and-scala-compatibility-notes)
-- [Troubleshooting](#troubleshooting)  
+  - [Compatible Scala and Play Framework versions](#compatible-scala-and-play-framework-versions)
+  - [Integrating with play-frontend-hmrc](#integrating-with-play-frontend-hmrc)
+  - [Finding Twirl templates for GOV.UK and HMRC design system components](#finding-twirl-templates-for-govuk-and-hmrc-design-system-components)
+  - [Using the components](#using-the-components)
+  - [Useful implicits](#useful-implicits)
+- [Creating HMRC-style pages](#creating-hmrc-style-pages)
+  - [Using the HMRC standard page template](#using-the-hmrc-standard-page-template)
+  - [Creating consistent page headings](#creating-consistent-page-headings)
+  - [Adding a sidebar to your Layout](#adding-a-sidebar-to-your-layout)
+- [Integrating with shared HMRC services](#integrating-with-shared-hmrc-services)
+  - [Adding a beta feedback banner](#adding-a-beta-feedback-banner)
+  - [Adding a User Research Banner](#adding-a-user-research-banner)
+  - [Linking to your accessibility statement](#linking-to-your-accessibility-statement)
+  - [Helping users report technical issues](#helping-users-report-technical-issues)
+  - [Allowing users to enable or disable tracking cookies](#allowing-users-to-enable-or-disable-tracking-cookies)
+- [Using common HMRC patterns](#using-common-hmrc-patterns)
+  - [Adding a dynamic character count to a text input](#adding-a-dynamic-character-count-to-a-text-input)
+  - [Adding accessible autocomplete to a select input](#adding-accessible-autocomplete-to-a-select-input)
+  - [Warning users before timing them out](#warning-users-before-timing-them-out)
+  - [Opening links in a new tab](#opening-links-in-a-new-tab)
+- [Advanced configuration](#advanced-configuration)
+  - [Adding your own SASS compilation pipeline](#adding-your-own-sass-compilation-pipeline)
+  - [Configuring non-HMRC projects to resolve play-frontend-hmrc artefacts](#configuring-non-hmrc-projects-to-resolve-play-frontend-hmrc-artefacts)
 - [Getting help](#getting-help)
-- [Resolve play-frontend-hmrc artefacts](#resolve-play-frontend-hmrc-artefacts)
-- [Useful links](#useful-links)
-- [Owning team readme](#owning-team-readme)
+  - [Troubleshooting](#troubleshooting)
+  - [Useful Links](#useful-links)
+- [Owning team README](#owning-team-readme)
 - [License](#license)
 
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
 ## Getting started
-1.  Add the version of [play-frontend-hmrc](https://github.com/hmrc/play-frontend-hmrc/releases) specific to your Play version
+
+### Compatible Scala and Play Framework versions
+
+This library is currently compatible with:
+* Scala 2.12 / 2.13
+* Play 2.8
```

Comment:
small one but in the regrouping I realised we don't mention the changelog. I know it's a pretty standard pattern but I think it's worth mentioning - especially with our major versions

```suggestion
* Play 2.8

### Finding out about library changes between versions

We summarise what's changed between versions, and importantly any actions that may be required when upgrading past a specific version within our [changelog](CHANGELOG.md).
```